packages/wallet-service/serverless.yml

```diff
@@ -324,10 +324,6 @@ functions:
           method: get
           cors: true
           authorizer: ${self:custom.authorizer.walletBearer}
-          request:
-            parameters:
-              paths:
-                index: false
```

Comment:
We would only need this block if we wanted to enforce the presence of a specific query in api gateway, which is not the case